tests/Fixtures/e2e/Unconfigured/run_tests.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+run () {
+    local INFECTION=${1}
+    local PHPARGS=${2}
```

Comment:
I'm not sure it's needed here. Probably copy pasted?

Reply:
Yes, it is, but I decided to leave this function `run` as it is because it should go to a common with all tests library. With these left you can see that it is [the same function as in e.g. here](https://github.com/infection/infection/blob/master/tests/Fixtures/e2e/Memory_Limit/run_tests.bash#L5), so you would change this file to use the common library without any second thoughts. 

If that's unacceptable, that's not problem, I'll remove these.

Reply:
thanks, that's ok